changelog/undistributed/changelog_show_bgp_20220307212947.rst

```diff
@@ -0,0 +1,7 @@
+--------------------------------------------------------------------------------
+                            Fix
+--------------------------------------------------------------------------------
+* IOSXE
+    * Modified ShowBgpNeighbor:
+        * Updated parsing to support VRF in bgp neighbors cli command instead of always setting 'default' VRF (parser p2_3)
+        * Added another files for unit testing (show ip bgp neighbors all, show bgp neighbors all)
```

Comment:
```suggestion
```

Reply:
Hi Lukas,
I added commit 22f421fe26fb7c3e226aea5c40b87c34f552d222 to remove the line from changelog